docs/words.txt

```diff
@@ -23,6 +23,8 @@ Redis
 serializable
 serialized
 serializer
+subreddit
```

Comment:
Is there a different place I should add them?   `subreddits` was actually triggering the spell check.